### DISCUSSION TITLE
depends on go-jsonrpc v0.1.4 version has bug.

### DISCUSSION THREAD

**Discussion by maweis1981** [filecoin-project/lotus#7351] [Ideas] [answered]
https://github.com/filecoin-project/lotus/blob/15d90c24edd3722b71df0b3828667ffde1982d3b/go.mod#L40

go-jsonrpc v0.1.4 told DO NOT USE.

v0.1.4
@raulk raulk released this on 19 Aug · 2 commits to master since this release

DO NOT USE -- this release is buggy.
Bug was introduced in #60, and solved in #63.
This release was never adopted (to our knowledge).

`https://github.com/filecoin-project/go-jsonrpc/releases/tag/v0.1.4`
[https://github.com/filecoin-project/go-jsonrpc/releases/tag/v0.1.4](https://github.com/filecoin-project/go-jsonrpc/releases/tag/v0.1.4)

**Comment by raulk** [comment, marked as answer]
```
github.com/filecoin-project/go-jsonrpc v0.1.4-0.20210217175800-45ea43ac2bec
```

is not depending on the buggy 0.1.4, it's depending on a commit after that.

We already have https://github.com/filecoin-project/go-jsonrpc/releases/tag/v0.1.5. Should probably adopt.
